openshift/cronJob-template.yaml

```diff
@@ -160,4 +160,10 @@ objects:
                     limits:
                       memory: "256Mi"
                       cpu: "500m"
+                  livenessProbe:
+                    failureThreshold: 1
+                    initialDelaySeconds: 1800
```

Comment:
so this is the maximum amount of time we grant the cronjob to consume?

Reply:
It *has to* finish in 30 minutes, if not, there is something fishy -> restart it. Most of the time I observed termination not being done during pushes of metrics to the push gateway.